src/App.jsx

```diff
@@ -1,104 +1,20 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const posts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId) || null,
```

Comment:
no needs to return `null` since if you don't find user you needed you'll get falsy value anyway
```suggestion
  user: usersFromServer.find(user => user.id === post.userId),
```